README.md

```diff
@@ -1,56 +1,25 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Kelvin Bush Portfolio
 
-> Description the project.
+> This is a simple website to show-case my skills as a software developer.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+- HTML
+- CSS
 
+## Live Demo (Coming Soon...)
 
 
 ## Authors
```

Comment:
- [ ] OPTIONAL: I recommend you adding setup section in readme to make it look more professional: 

Try this: 

```
 ## Getting Started

To get a local copy up and running follow these simple example steps.

- copy this link `https://github.com/kelvinbush/kelvin-portfolio.git`.
- get the directory that you want to clone the repository.
- open the command prompt in this directory.
- write `git clone https://github.com/kelvinbush/kelvin-portfolio.git`.
- go to the repository folder in your command prompt `cd project-capstone`
```